test/lib/tpunit++.cpp

```diff
@@ -93,6 +93,27 @@ int tpunit::TestFixture::tpunit_detail_do_run(int threads, std::function<void()>
     return tpunit_detail_do_run(include, exclude, before, after, threads, threadInitFunction);
 }
 
+void tpunit::TestFixture::tpunit_run_test_class(TestFixture* f) {
+   f->_stats._assertions = 0;
+   f->_stats._exceptions = 0;
+   tpunit_detail_do_methods(f->_before_classes);
+   if (f->_stats._assertions || f->_stats._exceptions) {
+      tpunit_detail_stats()._failures++;
+      tpunit_detail_stats()._failureNames.emplace(f->_name + "::BEFORE_CLASS"s);
+      cout << "\xE2\x9D\x8C !FAILED! \xE2\x9D\x8C initializing " << f->_name << ". Skipping tests." << endl;
```

Comment:
Do we still need both of these `cout`s?

Reply:
All the other (non-setup/teardown) tests print out when they fail, this just matches those.